src/desktop/color.rs

```diff
@@ -13,6 +13,10 @@ pub struct Color {
 }
 
 impl Color {
+    pub(crate) fn new(r: f64, g: f64, b: f64) -> Self {
+        Self { color: [r, g, b] }
+    }
```

Comment:
```suggestion
    pub(crate) fn from_vec(color: Vec<f64>) -> Self {
        debug_assert_eq!(color.len(), 3);
        Self {
            color: <[f64; 3]>::try_from(color).unwrap(),
        }
    }
```

Reply:
Thanks for your suggestions and for reviewing the changes. I really just don't understand though, how (ddd) is translated to [f64;3]. When I build a `zvariant::Value` from a `Vec<f64>`, it is converted to a `zvariant::Value::Array` with the signature `ad` instead of `(ddd)`.

Reply:
Oh indeed, because arrays are structs in dbus/serde world... annoying

Reply:
Also, I think if the screenshot color is also supposed to be `(ddd)` in the dict value, that the `Color` type should be (f64, f64, f64) internally